charts/kong/templates/_helpers.tpl

```diff
@@ -1488,6 +1488,13 @@ Kubernetes Cluster-scoped resources it uses to build Kong configuration.
   - get
   - patch
   - update
+- apiGroups:
```

Comment:
We can make this conditional on the version of KIC being deployed, right? No need to attach more policy rules than required.